### DISCUSSION TITLE
Working together?

### DISCUSSION THREAD

**Discussion by sevenrats** [DonutWare/Fladder#45] [General]
Are you aware that there is already a Flutter/Media-Kit jellyfin client?
https://github.com/jellyflix-app/jellyflix
It seems a shame not to be working on the same project. Are you open to working together?
Either way, you may be interested in our API package:
https://github.com/TentacleOrg/Tentacle

**Comment by sevenrats** [comment]
imho there are some great parts of this that should definitely be incorporated into jellyflix.  your library views and some aspects of the nav bar for example.  the main limitation of jellyflix right now is the fact that all libraries are lumped together by media type (i know.) but honestly I see the synthesis of these two apps as the ideal situation. some aspects of your nav bar are also superior we would want to make sure that a final output included those as well.
here is the current state of that work on our end:
https://github.com/jellyflix-app/jellyflix/pull/101

**Comment by PartyDonut** [comment]
Hey, thanks for reaching out! It's awesome that we both had similar ideas—and it’s always nice to see a fellow Riverpod fan! Right now, I’m focused on this project, especially since I’m really enjoying working on it.

I took a quick look at your Tentacle package, and it looks great! It would be quite a refactor for me to switch from Chopper, though.

So in short I would personally like to focus on this project, not just from a programming perspective but also from a design perspective.

Anyone can feel free to work on either project that's entirely up to them 😄.

**Reply by sevenrats** [reply]
No hard feelings I totally get that. At a minimum I’d like to steal your library views code and hit you up occasionally when I get stuck. Icanhazdat?

**Reply by PartyDonut** [reply]
Absolutely, it's open-source, so you can implement it however you like! If you use large parts of it, though, a small attribution would be appreciated.
